spring-integration-hazelcast/build.gradle

```diff
@@ -45,7 +45,7 @@ configurations {
 
 dependencies {
 	compile "org.springframework.integration:spring-integration-core:$springIntegrationVersion"
-	compile "com.hazelcast:hazelcast:$hazelcastVersion"
+	compile "com.hazelcast:hazelcast-all:$hazelcastVersion"
```

Comment:
M-m-m. Can we don't use `-all`? Would be better to add one more required dependency, but as optional and let end-user to choose.
The main reason, that not all application will be interested in the full-featured support from us